<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Principal</class>
 <widget class="QMainWindow" name="Tela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 188, 186);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:10px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:10px
}</string>
    </property>
    <property name="text">
     <string>Cadastrar Veículos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:10px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:10px
}

gridline-color: rgb(156, 255, 209);</string>
    </property>
    <property name="text">
     <string>Cadastrar  Usuário</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:10px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:10px
}</string>
    </property>
    <property name="text">
     <string>Sistema de Controle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:10px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:10px
}</string>
    </property>
    <property name="text">
     <string>Cadastrar Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:10px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:10px
}</string>
    </property>
    <property name="text">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>540</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(112, 133, 145);
	border-radius:15px
} 

QPushButton:hover{

	color:rgb(112, 133, 145); 
	background-color:rgb(255, 255, 255);
	border-radius:15px
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1441</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>bmw-z3-mean-1549766.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1396</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton_</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
